### PR TITLE
feat: `--uninstall` switch & move unmount script to command

### PR DESCRIPTION
**Pull request by Itroublve**
This PR adds a uninstall switch to uninstall ReVanced apps and moves the unmount script file to a variable.

### CODE REVIEW

**Comment by Itroublve** [comment]
Uh, a temporary workaround for now. I open a pr for patcher tomorrow unless someone else wants to do it c:

**Comment by Itroublve** [comment]
Considering the patcher needs updating, I'll leave it for later when abstract options is added. For now this workaround works just OK and shall be updated later.

**Comment by oSumAtrIX** [comment]
Before merging this, can you open an issue to abstract the patcher methods to fix the temporary solution used here?

**Comment by Itroublve** [comment]
> Before merging this, can you open an issue to abstract the patcher methods to fix the temporary solution used here?

In this repo or the patcher repo?

**Comment by oSumAtrIX** [comment]
The patcher repo

File: src/main/kotlin/app/revanced/cli/command/MainCommand.kt [hunk 1]
```diff
@@ -33,6 +33,20 @@ internal object MainCommand : Runnable {
     lateinit var args: Args
 
     class Args {
+        @Option(names = ["-a", "--apk"], description = ["Input file to be patched"], required = true)
+        lateinit var inputFile: File
+
+        @Option(names = ["--uninstall"], description = ["Completely uninstall root variant"])
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/cli/command/MainCommand.kt:39]:
```suggestion
        @Option(names = ["--uninstall"], description = ["Uninstall the mount variant"])
```

**Pull request merged**